jsontext/decode.go

```diff
@@ -400,6 +400,36 @@ func (d *decoderState) SkipValue() error {
 	}
 }
 
+func (d *Decoder) SkipValueRemainder() error {
+	return d.s.SkipValueRemainder() // MUSTDO: Remove
```

Comment:
did you mean to resolve this before sending the PR? I've never seen a MUSTDO before, is it like a TODO which we can leave here, or is it like a XXX that must be resolved before merging?

Reply:
Ooops. This is supposed to be my equivalent of a "DO NOT SUBMIT" marker.

Reply:
Oh, it's already gone on latest HEAD.

Reply:
Ah, I started reviewing before one of your last pushes, I think.